classes/PaymentModule.php

```diff
@@ -519,8 +519,6 @@ public function validateOrder(
             $old_message = Message::getMessageByCartId((int) $this->context->cart->id);
             if ($old_message && !$old_message['private']) {
                 $update_message = new Message((int) $old_message['id_message']);
```

Comment:
I guess this is useless now? You can use `$old_message->message` to populate `$customer_thread->message`